apps/vaults/hooks/usePortalsApi.ts

```diff
@@ -96,67 +95,46 @@ const NETWORK = new Map<number, string>([
 	[56, 'bsc']
 ]);
 
-type TUsePortalsApi = {
-	getEstimate: (props: TGetEstimateProps) => Promise<TPortalEstimate | null>;
-	getTransaction: (
-		props: TGetTransactionProps
-	) => Promise<TPortalTransaction | null>;
-	getApproval: (props: TGetApprovalProps) => Promise<TPortalsApproval | null>;
-	error: unknown;
-};
+type TPortalsEstimateResp = {
+	data: TPortalEstimate | undefined,
+	error?: Error | undefined
+}
+export async function getPortalsEstimate({network, params}: TGetEstimateProps): Promise<TPortalsEstimateResp> {
+	const baseURL = 'https://api.portals.fi/v1';
+	const axiosInstance: AxiosInstance = axios.create({baseURL});
 
-const usePortalsApi = (): TUsePortalsApi => {
-	const [error, set_error] = useState<unknown | null>(null);
+	try {
+		const endpoint = `/portal/${NETWORK.get(network)}/estimate`;
```

Comment:
This should be called `path` as endpoint would be the whole thing
```suggestion
		const path = `/portal/${NETWORK.get(network)}/estimate`;
```